sklearn/feature_extraction/tests/test_text.py

```diff
@@ -324,6 +324,23 @@ def test_tf_idf_smoothing():
     assert_true((tfidf >= 0).all())
 
 
+def test_tfidf_partial_fit():
+    X = [[1, 1, 1],
+         [1, 1, 0],
+         [1, 0, 0]]
+
+    tr_full = TfidfTransformer(smooth_idf=True, norm='l2')
+    tfidf_full = tr_full.fit_transform(X).toarray()
+
+    tr_partial = TfidfTransformer(smooth_idf=True, norm='l2')
+    tr_partial.fit([[1, 1, 1]])
```

Comment:
no, we'd usually use `partial_fit` all the way through, not `fit` the first time.

Reply:
@n0mad When calling `partial_fit` all the way through, the use case would be to provide a fixed vocabulary at initialization (otherwise it wouldn't make sense indeed). 

Generally, to do out-of-core text vectorization one would do the first pass on the dataset to estimate the vocabulary (but currently there is no way of doing that in scikit-learn without also computing the full Bag of Words matrix), then a second pass to actually compute the BoW matrix (though that's probably not compatible with sklearn API). Gensim does something like that I think, but I can't find the exact reference anymore...